auth_otp/views.py

```
@@ -0,0 +1,357 @@
+        
+        # ایجاد یا دریافت کاربر
+        user, is_new = AuthService.create_user_if_not_exists(
+            phone_number=otp_request.phone_number,
+            user_type='patient'  # نوع پیش‌فرض
+        )
+        
+        # تولید توکن‌ها
+        tokens = AuthService.generate_tokens(user)
+        
+        # ایجاد رکورد verification
+        verification = AuthService.create_verification_record(
```

Ruff (0.12.2)

Local variable `verification` is assigned to but never used

Remove assignment to unused variable `verification`

(F841)